cdap-ui/app/directives/widget-container/widget-input-schema/widget-input-schema.html

```diff
@@ -0,0 +1,183 @@
+            </th>
+            <th>
+              <label class="control-label">Null</label>
+            </th>
+          </thead>
+          <!--
+            This is bad. If we change the container's css class then infinite scroll is not going to work.
+            The reason we are doing this today is because we need to have one scroll bar for all the three
+            sections of the node config modal (input-schema, config & output-schema) and that is technically
+            the container for the inifinite scroll here.
+
+            So container = bottompanel-body & element = table/tbody. As we scroll in the container
+            the ng-inifinite scroll is going to check whether the difference between element bottom & container
+            bottom is less than the scroll distance, if yes then its going to add additional elements.
+
+            The damage is restricted to the css classes we add to the modal and the uibModal service (windowClass)
+          -->
+          <tbody infinite-scroll="MyInputSchemaCtrl.doLoadNextSetOfInputSchemaRows()"
+                 infinite-scroll-container="'.modal.hydrator-modal.node-config-modal .bottompanel-body'"
+                 infinite-scroll-disabled="inputschema.schema.fields <= MyInputSchemaCtrl.inputSchemaRowLimit">
+            <tr ng-repeat="field in ::inputschema.schema.fields | limitTo: 10 track by $index">
+              <td class="name">
+                <span uib-tooltip="{{ ::field.name }}"
+                      tooltip-ellipsis="{{ ::field.name }}"
+                      data-ellipsis="isEllipsis"
+                      data-offset-value="20"
+                      tooltip-enable="isEllipsis"
+                      tooltip-append-to-body="true">
+                  {{::field.name}}
+                </span>
+              </td>
+              <td class="type">
+                <span ng-if="field.type.type !== 'map'">{{::field.type}}</span>
+                <span ng-if="field.type.type === 'map'">map&lt;string, string&gt; </span>
+              </td>
+              <td class="nullable text-center" >
+                <input type="checkbox" ng-model="field.nullable" disabled="disabled">
+              </td>
+            </tr>
+          </tbody>
+        </table>
+
+        <div ng-if="::(inputschema.schema.fields.length === 0)">
+          <p>There is no input schema</p>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <!-- End of section that shows multiple inputs -->
+
+  <!-- Section that shows single input schema-->
+
+  <div ng-if="::(!MyInputSchemaCtrl.multipleInputs)">
+    <!-- Section that shows single input schema table-->
+
+    <table class="table"
+      ng-if="::(MyInputSchemaCtrl.inputSchemas[0].schema.fields.length > 0)">
+      <thead>
+        <th>
+          <label class="control-label">Name</label>
+        </th>
+        <th>
+          <label class="control-label">Type</label>
+        </th>
+        <th>
+          <label class="control-label">Null</label>
```

Comment:
This is what we have currently across both input & output schemas. We have it this way because we needed the headings to be bold + some styling which we should cleanit up as a whole. Not sure why we have a label inside `th` element.